pyproject.toml

```diff
@@ -56,8 +57,27 @@ ignore = [
     "A",
     # Print statements.
     "T20",
+    "ERA001", # The "commented out code issue
+    "PTH123", # i do not want to clutter the dom space with more things
+    "ASYNC230", # anyio is not supported in this, maybe
     # TODOs.
     "TD002",
     "TD003",
     "FIX",
 ]
+
+[tool.pixi.workspace]
+channels = ["conda-forge"]
+platforms = ["linux-64"]
+
+[tool.pixi.pypi-dependencies]
+code-jam-template = { path = ".", editable = true }
+
+[tool.pixi.environments]
+default = { solve-group = "default" }
+dev = { features = ["dev"], solve-group = "default" }
+
+[tool.pixi.tasks]
+
+[tool.pixi.dependencies]
+pre-commit = ">=4.2.0,<5"
```

Comment:
Not sure you'll need to do this? I set up things on `main` s.t. requirements are put in the dependency group above then placed in `requirements.txt`.